### ISSUE TITLE
Use some Material Docs features to make the docs look better.

### ISSUE THREAD

**Issue by kujtimiihoxha** [kujtimiihoxha/iris-docs#2]
@kataras I think we should use material docs features where we can to make the docs more readable. 
Like  ex. Notes and warnings 
![capture](https://cloud.githubusercontent.com/assets/14311743/17243202/a7516a02-557b-11e6-9079-40600c27c00e.PNG)

Would it be OK to group the content in some sections so the sidebar gets smaller, I think that how it is done now it look to overwhelming and maybe you could give me some ideas on how the grouping would be done.

Thanks for the help ;)

**Comment by ghost** [comment]
This will be nice but it will take time to do, keep that as the last-thing todo before publishing?

**Comment by kujtimiihoxha** [comment]
@kataras What about the sections on the sidebar ? I need to know what to do now.

**Comment by ghost** [comment]
What do you mean? We miss/forgot some sections, gitbook has more,

**Comment by kujtimiihoxha** [comment]
> Would it be OK to group the content in some sections so the sidebar gets smaller, I think that how it is done now it look to overwhelming and maybe you could give me some ideas on how the grouping would be done.

I mean this.

**Comment by ghost** [comment]
Hmm I think the group is almost good, the other solution  is to group them by `context` functions `iris station's` functionalities and `global iris` functionality but I don't know if that is better than we have now. But you can change them do whatever you're thinking, you're totally free, do the thing you imagine and if you don't like just restore a backup

**Comment by kujtimiihoxha** [comment]
No problem, I will finish the other sections as in the gitbook, I just thought that it is a rather big sidebar but OK I will tell you when its finished ;)

**Comment by ghost** [comment]
Yes, finish the basics first and do changes later I don't have the best design-eye, so you know better :P

**Comment by kujtimiihoxha** [comment]
@kataras I think all the links are finished now. All that remains is that : 
- You check if I missed something.
- Tell me if you want to change the color scheme to match iris.[these are the colors available](http://themes.gohugo.io/theme/material-docs/getting-started/#changing-the-color-palette)

If all is good I will build the static files and push them to iris-contrib/website

**Comment by ghost** [comment]
Ok,

> You check if I missed something.

Table of contents  | broken links
Routing and reverse lookups | http://localhost:1313/routing/
Send e-mails |  http://localhost:1313/mail
Body binder |  http://localhost:1313/request_body_bind/
Examples redirects to http://localhost:1313/examples instead of https://github.com/iris-contrib/examples

> Tell me if you want to change the color scheme to match iris.these are the colors available

I would like to use the third color pallete ( from the left-ro-right [screen](http://themes.gohugo.io/theme/material-docs/images/colors.png))

**Comment by kujtimiihoxha** [comment]
@kataras all done :smile:

**Comment by kataras** [comment]
Nice!!!, I don't see the changes on the repo or via pr on iris-contrib/website, do your own things ( group sections or fix english or whatever you thinking of to make it nicer) and pr it!!!!
